.release/release-metadata.hcl

```diff
@@ -0,0 +1,6 @@
+url_docker_registry_dockerhub = "https://hub.docker.com/r/hashicorp/vault-plugin-auth-azure"
+url_docker_registry_ecr       = "https://gallery.ecr.aws/hashicorp/vault-plugin-auth-azure"
```

Comment:
I think we can omit these since there's no image being built. At least that seems to be ok in VLE: https://github.com/hashicorp/vault-lambda-extension/blob/main/.release/release-metadata.hcl

Reply:
:+1: